common/rdm/RDMAPI.cpp

```diff
@@ -2794,7 +2871,7 @@ bool RDMAPI::SetDimmerMinimumLevels(
   struct minimum_levels {
     uint16_t min_increasing;
     uint16_t min_decreasing;
-    uint8_t on_below_min;
```

Comment:
I think this should stay as uint8_t in the struct, just to avoid any oddness with some theoretical future OS doing "smart" things. You should be able to coerce into it okay.

Reply:
Good call. Works fine without any explicit casting going on.

Reply:
Done